packages/common/src/api/authorizedApps.ts

```diff
@@ -37,7 +38,7 @@ const authorizedAppsApi = createApi({
     removeAuthorizedApp: {
       async fetch(args: RemoveAuthorizedAppArgs, { audiusSdk }) {
         const { userId, apiKey } = args
-        const encodedUserId = encodeHashId(userId) as string
+        const encodedUserId = Id.parse(userId)
```

Comment:
And as a counter-example, throwing during the fetch part of a request (not during the response adapters) is generally safe because the request would fail anyway.